src/COMPONENTS/HEADER/Header.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import setting from "../../images/settingIcon.png";
+import microphone from "../../images/mic.png";
+
+
+const Header = () => {
+    return (
+      <header>
+        <Link className="Header" to="/">
+          <h1 className="logoname">Celestial Store</h1>
+          <div className="headerIcons">
+            <img className="setting" src={setting} alt="setting" />
+            <img className="mic" src={microphone} alt="micIcon" />
+          </div>
+        </Link>
+      </header>
```

Comment:
For the Header design:

- [ ] Please adjust the height of the header and make it smaller.
- [ ] Please make the font of `h1` smaller and place it in the middle.
- [ ] Kindly change the icons color to make it appear properly to match the given design.

- [ ] and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/189374743-dda28b48-2a04-4d72-a93b-5d52b7074535.png)  | ![image](https://user-images.githubusercontent.com/68796232/189374492-70ce4ee3-21d5-4af5-abba-cd81bf91a800.png)  |